### PR TITLE
adds logs guidance for new bug reports

### PR DESCRIPTION
**Pull request by brennanMKE**
*Issue #, if available:*

*Description of changes:*

*Check points: (check or cross out if not relevant)*

- [ ] Added new tests to cover change, if needed
- [ ] Build succeeds with all target using Swift Package Manager
- [ ] All unit tests pass
- [ ] All integration tests pass
- [ ] Security oriented best practices and standards are followed (e.g. using input sanitization, principle of least privilege, etc)
- [ ] Documentation update for the change if required
- [ ] PR title conforms to conventional commit style
- [ ] If breaking change, documentation/changelog update with migration instructions

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yaml [hunk 1]
```diff
@@ -92,7 +92,15 @@ body:
       label: Relevant log output
       description: >-
         Include any relevant log output under
-        `~/.amplify/logs/amplify-cli-<issue-date>.log`
+        `~/.amplify/logs/amplify-cli-<issue-date>.log` (See: [Logs Guidance](https://github.com/aws-amplify/amplify-ios/blob/main/LogsGuidance.md.md))
```
Comment (harsh62) [inline .github/ISSUE_TEMPLATE/bug_report.yaml:95]: Is this link correct?

Comment (brennanMKE) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:95]: It will be once it is merged to main.

Comment (brennanMKE) [reply .github/ISSUE_TEMPLATE/bug_report.yaml:95]: Wait, extra `.md`

File: LogsGuidance.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+# Logs Guidance
+
+When providing logs in an issue please following these recommendations.
```
Comment (harsh62) [inline LogsGuidance.md:3]: nitpick: `following` -> `follow`

**Review by harsh62 — APPROVED** [review]

**Pull request merged**